bom/pom.xml

```diff
@@ -60,8 +60,8 @@
     <mockito.version>3.7.7</mockito.version>
     <mongodb-image.name>mongo:4.2.13</mongodb-image.name>
     <mongodb-driver.version>3.10.1</mongodb-driver.version>
-    <netty.version>4.1.49.Final</netty.version>
-    <netty.tcnative.version>2.0.30.Final</netty.tcnative.version>
+    <netty.version>4.1.60.Final</netty.version>
```

Comment:
The netty update seems to cause such an exception when building the Quarkus native images:
````
[hono-adapter-amqp-vertx-quarkus-1.8.0-SNAPSHOT-runner:25]    classlist:   9.339,38 ms,  1,19 GB
[hono-adapter-amqp-vertx-quarkus-1.8.0-SNAPSHOT-runner:25]        setup:     944,32 ms,  1,19 GB
Error: Could not find target method: static io.netty.handler.ssl.SslContext io.quarkus.netty.runtime.graal.Target_io_netty_handler_ssl_SslContext.newServerContextInternal(io.netty.handler.ssl.SslProvider,java.security.Provider,java.security.cert.X509Certificate[],javax.net.ssl.TrustManagerFactory,java.security.cert.X509Certificate[],java.security.PrivateKey,java.lang.String,javax.net.ssl.KeyManagerFactory,java.lang.Iterable,io.netty.handler.ssl.CipherSuiteFilter,io.netty.handler.ssl.ApplicationProtocolConfig,long,long,io.netty.handler.ssl.ClientAuth,java.lang.String[],boolean,boolean,java.lang.String) throws javax.net.ssl.SSLException
com.oracle.svm.core.util.UserError$UserException: Could not find target method: static io.netty.handler.ssl.SslContext io.quarkus.netty.runtime.graal.Target_io_netty_handler_ssl_SslContext.newServerContextInternal(io.netty.handler.ssl.SslProvider,java.security.Provider,java.security.cert.X509Certificate[],javax.net.ssl.TrustManagerFactory,java.security.cert.X509Certificate[],java.security.PrivateKey,java.lang.String,javax.net.ssl.KeyManagerFactory,java.lang.Iterable,io.netty.handler.ssl.CipherSuiteFilter,io.netty.handler.ssl.ApplicationProtocolConfig,long,long,io.netty.handler.ssl.ClientAuth,java.lang.String[],boolean,boolean,java.lang.String) throws javax.net.ssl.SSLException
	at com.oracle.svm.core.util.UserError.abort(UserError.java:68)
	at com.oracle.svm.hosted.substitute.AnnotationSubstitutionProcessor.findOriginalMethod(AnnotationSubstitutionProcessor.java:711)
[...]
````
I guess we have to wait for an updated Quarkus release that uses vert.x 3.9.7 and the above netty version for this to work.
The current Quarkus 1.13.3.Final version only uses vert.x 3.9.6.

Reply:
I see. That seems to be an issue with Netty, though. The signature of (package private) method `io.netty.handler.ssl.SslContext.newServerContextInternal` has changed between 4.1.49 and 4.1.60.
Another example for why you should be very careful when using reflection to invoke methods that you are not supposed to invoke in the first place :-)
I'll take a look and see if we can maybe fall back to the latest Netty version with the original method signature or maybe we can even remove *netty-tcnative* from the native image build altogether as we do not use a native implementation anyway but use the SslContext only to check if there is a native TLS implementation available or not ...